lms/djangoapps/teams/static/teams/js/spec_helpers/team_spec_helpers.js

```diff
@@ -124,10 +216,17 @@ define([
     return {
         testCourseID: testCourseID,
         testUser: testUser,
+        testCountries: testCountries,
+        testLanguages: testLanguages,
         testTeamDiscussionID: testTeamDiscussionID,
+        createMockTeamData: createMockTeamData,
+        createMockTeams: createMockTeams,
+        createMockTeamMembershipsData: createMockTeamMembershipsData,
+        createMockTeamMemberships: createMockTeamMemberships,
         createMockPostResponse: createMockPostResponse,
         createMockDiscussionResponse: createMockDiscussionResponse,
         createAnnotatedContentInfo: createAnnotatedContentInfo,
-        createMockThreadResponse: createMockThreadResponse
+        createMockThreadResponse: createMockThreadResponse,
+        verifyCards: verifyCards
     };
```

Comment:
I really dislike this object; can we change to returning the values directly instead of duplicating the variable names and property names? Some functions, like `createMockTeamData`, would need to change from `var createMockTeamData = function (...) { ... }` to `{ createMockTeamData: function createMockTeamData ( ... ) { ... }}`.